libraries/AP_Motors/AP_MotorsHeli_RSC.h

```diff
@@ -178,16 +181,18 @@ class RSCGovFloatParam {
     static const struct AP_Param::GroupInfo var_info[];
 
     void set_gov_enable(int8_t setenable) {enable = setenable; }
-    int16_t get_setpoint() { return setpoint; }
+    int16_t get_reference() { return reference; }
+    float get_range() { return range; }
     float get_disengage() { return disengage; }
-    float get_droop_setting() { return droop_setting; }
+    float get_droop_response() { return droop_response; }
```

Comment:
I wonder if this "response" is really a "gain"?

Reply:
Not really. It is the governor's droop response setting as a percentage of the max allowable droop. It could be considered a type of gain, but since this is a droop speed control, and not a PID control, the droop setting determines the governor's response to the fuel control. It is the droop setting that moves the governor's reference speed and determines the governor's stability without going out on over-speed.

see (on page 5):http://www.woodward.com/WorkArea/DownloadAsset.aspx?id=2147483661

I elected to use this type of speed control over a PID due to the droop governor's fantastic stability without surging. Since it is a different type of design that commonly seen in ArduPilot some of the terminology used will be a bit confusing or different than a typical PID control.